### DISCUSSION TITLE
Please help me to change YouTube API automatically when quota exceeded.

### DISCUSSION THREAD

**Discussion by arshalabbas** [eritislami/evobot#359] [Q&A]
I need to change the Youtube API automatically when the quota exceeded. I got an idea but I don't know how to use it, because I'm a beginner in coding.
The idea is when the error came that the youtube API quota exceeded, need to change "YOUTUBE_API_KEY" to "YOUTUBE_API_KEY2" from the config file.
maybe it will work but I don't know to do it, I hope that anyone will help me :)

**Comment by BlazeIsClone** [comment]
yup I was also trying to figure this out I couldn't find a way to catch that specific err to do it.

**Reply by arshalabbas** [reply]
😣
